fuas/meta.py

```diff
@@ -35,20 +35,11 @@ def keepname():
         os.remove(conf.namefile)
     with open(conf.namefile, "a") as file:
         nmls = []
-        resp = conf.cobj.list_users(page_size=conf.dfltsize)
-        rslt = resp.result
-        for indx in rslt:
-            nmls.append(indx["username"])
+        for user in conf.cobj.list_all_entities("users", page_size=conf.dfltsize):
+            print(user)
+            nmls.append(user["username"])
             lqnt += 1
-        file.write("\n".join(nmls) + "\n")
-        iter = (conf.totlqant // conf.dfltsize) - 1
-        for indx in range(iter):
-            rslt = resp.next_page().result
-            nmls = []
-            for indx in rslt:
-                nmls.append(indx["username"])
-                lqnt += 1
-            file.write("\n".join(nmls) + "\n")
+            file.write(f"{user['username']}\n")
```

Comment:
Is this intended to make a write once in every iteration or should the writing of the usernames be beyond the scope of the loop, as I mentioned below?

```suggestion
        file.write(f"{user['username']}\n")
```

Reply:
Ok no. The file was opened on `append` mode so writing to the file on every iteration is important.